app/views/sandbox/budget_guide.html.erb

```diff
@@ -0,0 +1,269 @@
+<div class="column">
+
+  <main class="content">
+    <header class="gradient-bg">
+      <div class="inner">
+        <h1>Así funciona un presupuesto municipal</h1>
+        <p>El presupuesto municipal es el documento que explica cómo se gestiona el dinero público durante un año. Aquí te explicamos los conceptos más importantes para que no te pierdas en su lectura.</p>
+
+        <ul>
```

Comment:
This is the TOC (table of contents). Should we give it a special/different format?